spec/spec_helper_acceptance.rb

```diff
@@ -11,6 +11,22 @@
   set :backend, :exec
 end
 
+module Specinfra
+  module Backend
+    class Cmd
+      def execute_script(script)
+        if Open3.respond_to?(:capture3)
```

Comment:
That's a copy of the upstream method, just with the return value munging removed. I'm guessing specinfra supports some old version of ruby where open3 doesn't have a capture3 method? Seems odd, but it works *shrug*